Kickstarter-iOS/Features/SystemDesign/Colors/SystemDesignColors.swift

```diff
@@ -0,0 +1,197 @@
+import Prelude
+import Prelude_UIKit
+import UIKit
+
+public enum SystemDesignColorStyle: String, CaseIterable {
+  // MARK: - Greens
+
+  case create100
+  case create300
+  case create500
+  case create700
+
+  // MARK: - Greys
+
+  case black
+  case support100
+  case support200
+  case support300
+  case support400
+  case support500
+  case support700
+  case white
+
+  // MARK: - Blues
+
+  case trust100
+  case trust300
+  case trust500
+  case trust700
+
+  // MARK: - Corals
+
+  case celebrate100
+  case celebrate300
+  case celebrate500
+  case celebrate700
+
+  // MARK: - Functional
+
+  case alert
+  case cellSeparator
+  case facebookBlue
+  case inform
+  case warn
+}
+
+extension SystemDesignColorStyle {
+  public func load() -> UIColor? {
+    UIColor(named: self.rawValue)
+  }
+}
+
+public func adaptiveColor(_ style: SystemDesignColorStyle) -> UIColor {
+  style.load()! // not ideal to force unwrap. need to work on a better approach.
+}
+
+// MARK: - Green
+
+public let adaptiveGreenButtonStyle = baseButtonStyle
+  <> UIButton.lens.titleColor(for: .normal) .~ adaptiveColor(.white)
```

Comment:
We can't `greenButtonStyle` as is because it's using hardcoded hex values instead of these new Xcode color assets. updating that directly would impact the entire app. I'd rather stick to the demo page and start addressing the rest of the app separately.